manuscripts/report.py

```diff
@@ -630,9 +633,21 @@ def create_pdf(self):
         templates_path = os.path.join(os.path.dirname(__file__),
                                       "latex_template")
         copy_tree(templates_path, report_path)
+        # if user specified a logo then replace it with default logo
+        if self.logo:
+                os.remove(os.path.join(report_path, "logo.eps"))
+                print(copy_file(self.logo, os.path.join(report_path, "logo." + self.logo.split('/')[-1].split('.')[-1])))
         # Copy the data generated to be used in LaTeX template
-        copy_tree(self.data_dir, os.path.join(report_path, "data"))
-        copy_tree(self.data_dir, os.path.join(report_path, "figs"))
+        current_dir=os.getcwd()
```

Comment:
I had not realized this was about solving #24, but now I did after @acs comments there. Please, remove these changes from here, and open a new pull request for them, so that we can follow two separate reviews: one for the logo stuff (this one, which is almost done) and another one for the recursive copy (#24).